README.md

```diff
@@ -13,7 +13,7 @@ A Flutter plugin for Android and iOS supports cropping images
 
 ### Android
 
-- Add UCropActivity into your AndroidManifest.xml
+- Add UCropActivity into your AndroidManifest.xml **after** the Main Activity
```

Comment:
I think your wording is not correct. An activity must be placed inside `application` tag and it doesn't matter the order among `activity` tags.